data/biocypher_config.yaml

```diff
@@ -15,4 +15,11 @@ outdir: biocypher-out
 # Clear ontology cache
 # BioCypher caches the ontology scaffold for performance reasons
 # Set to true to clear the cache and re-download the ontology
-clear_cache: true
\ No newline at end of file
+clear_cache: true
+
+biocypher:
+  tail_ontologies:
```

Comment:
we will be adding the remaining ontologies as we are loading the datasets into the dev environment